scripts/prepare_pr_comment.py

```diff
@@ -1,13 +1,34 @@
+import os
 import sys
 
-def prepare_failure_commet(repository, issue_number, vendor_label, chart_name):
-    msg = f"Thank you for the PR #{issue_number}!\n\n"
-    msg += f"There are [some errors](https://github.com/{repository}/pull/{issue_number}/checks) while building and validating your PR. Please check the log for more details.\n\n"
-    msg += f'/metadata {{"vendor_label": "{vendor_label}", "chart_name": "{chart_name}"}}\n\n'
-    msg += f"Partner Support URL: https://redhat-connect.gitbook.io/red-hat-partner-connect-general-guide/managing-your-account/getting-help/technology-partner-success-desk\n"
```

Comment:
This messages do not match the test you posted here. Can you run the test again to see this message format?

Reply:
These red lines are removed. See below for new text.